plugins/arDominionB5Plugin/modules/sfIsaarPlugin/templates/indexSuccess.php

```diff
@@ -60,201 +60,209 @@
+      <?php echo link_to_if(QubitAcl::check($resource, 'update'), render_b5_section_label(__('Control area')), [$resource, 'module' => 'actor', 'action' => 'edit'], ['anchor' => 'control-collapse', 'title' => __('Edit control area'), 'class' => 'text-primary']); ?>
 
-  <?php echo render_show(__('Authority record identifier'), $resource->descriptionIdentifier); ?>
+      <?php echo render_show(__('Authority record identifier'), $resource->descriptionIdentifier); ?>
 
-  <?php if (null !== $repository = $resource->getMaintainingRepository()) { ?>
-    <?php echo render_show(__('Maintained by'), link_to(render_title($repository), [$repository, 'module' => 'repository'])); ?>
-  <?php } ?>
+      <?php if (null !== $repository = $resource->getMaintainingRepository()) { ?>
+        <?php echo render_show(__('Maintained by'), link_to(render_title($repository), [$repository, 'module' => 'repository'])); ?>
+      <?php } ?>
 
-  <?php echo render_show(__('Institution identifier'), render_value($resource->getInstitutionResponsibleIdentifier(['cultureFallback' => true]))); ?>
+      <?php echo render_show(__('Institution identifier'), render_value_inline($resource->getInstitutionResponsibleIdentifier(['cultureFallback' => true]))); ?>
 
-  <?php echo render_show(__('Rules and/or conventions used'), render_value($resource->getRules(['cultureFallback' => true]))); ?>
+      <?php echo render_show(__('Rules and/or conventions used'), render_value($resource->getRules(['cultureFallback' => true]))); ?>
 
-  <?php echo render_show(__('Status'), render_value($resource->descriptionStatus)); ?>
+      <?php echo render_show(__('Status'), render_value_inline($resource->descriptionStatus)); ?>
 
-  <?php echo render_show(__('Level of detail'), render_value($resource->descriptionDetail)); ?>
+      <?php echo render_show(__('Level of detail'), render_value_inline($resource->descriptionDetail)); ?>
 
-  <?php echo render_show(__('Dates of creation, revision and deletion'), render_value($resource->getRevisionHistory(['cultureFallback' => true]))); ?>
+      <?php echo render_show(__('Dates of creation, revision and deletion'), render_value($resource->getRevisionHistory(['cultureFallback' => true]))); ?>
 
-  <div class="field">
-    <h3><?php echo __('Language(s)'); ?></h3>
-    <div>
-      <ul>
-        <?php foreach ($resource->language as $code) { ?>
-          <li><?php echo format_language($code); ?></li>
-        <?php } ?>
-      </ul>
-    </div>
-  </div>
+      <div class="field <?php echo render_b5_show_field_css_classes(); ?>">
+        <?php echo render_b5_show_label(__('Language(s)')); ?>
+        <div class="<?php echo render_b5_show_value_css_classes(); ?>">
+          <ul class="<?php echo render_b5_show_list_css_classes(); ?>">
+            <?php foreach ($resource->language as $code) { ?>
+              <li><?php echo format_language($code); ?></li>
+            <?php } ?>
+          </ul>
+        </div>
+      </div>
```

Comment:
For this cases that are not a `string` or a call to `__toString()`, we could format the array in the template ...

```php
<?php
    $list = [];
    foreach ($resource->language as $code) {
        $list[] = format_language($code);
    }
    echo render_show(__('Language(s)'), $list);
?>
```

It may require further tweaks as we go, but this is what I was thinking yesterday.